src/jit/morph.cpp

```diff
@@ -1608,6 +1642,25 @@ void fgArgInfo::ArgsComplete()
                     // Spill multireg struct arguments that are expensive to evaluate twice
                     curArgTabEntry->needTmp = true;
                 }
+                else if (argx->OperGet() == GT_OBJ)
+                {
+                    GenTreeObj*           argObj     = argx->AsObj();
+                    CORINFO_CLASS_HANDLE  objClass   = argObj->gtClass;
+                    unsigned              structSize = compiler->info.compCompHnd->getClassSize(objClass);
+                    switch (structSize)
+                    {
+                    case 11:
+                    case 13:
+                    case 14:
+                    case 15:
```

Comment:
Please add a comment why we currently need a tmp.  Because RyuJit Arm64 back-end doesn't handle it currently?  Would it make sense to add a TODO-CQ comment along with a Git issue?